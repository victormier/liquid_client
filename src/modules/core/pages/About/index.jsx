import React from 'react';
import { Grid, PageHeader } from 'react-bootstrap';

const About = () => (
  <Grid>
    <PageHeader>React Redux Apollo GraphQL Hot Boilerplate</PageHeader>
  </Grid>
);

export default About;
