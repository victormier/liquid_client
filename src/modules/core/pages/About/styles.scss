.code_block {
  display: block;
  white-space: pre-wrap;
  padding: 10px 15px 10px 15px;
}

.todo_item {
  display: block;
}
