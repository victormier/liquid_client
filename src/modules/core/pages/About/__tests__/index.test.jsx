import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import About from '../index'


describe('component', () => {
  it('renders', () => {
    const container = shallow(
      <About />
    )

    expect(shallowToJson(container)).toMatchSnapshot()
  })
})
