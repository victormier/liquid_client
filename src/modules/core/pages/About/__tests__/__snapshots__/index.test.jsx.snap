// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders 1`] = `
<Grid
  bsClass="container"
  componentClass="div"
  fluid={false}
>
  <PageHeader
    bsClass="page-header"
  >
    React Redux Apollo GraphQL Hot Boilerplate
  </PageHeader>
</Grid>
`;
